litestar/cli/_utils.py

```diff
@@ -134,6 +136,8 @@ def from_env(cls, app_path: str | None, app_dir: Path | None = None) -> Litestar
         uds = getenv("LITESTAR_UNIX_DOMAIN_SOCKET")
         fd = getenv("LITESTAR_FILE_DESCRIPTOR")
         reload_dirs = tuple(s.strip() for s in getenv("LITESTAR_RELOAD_DIRS", "").split(",") if s) or None
+        reload_includes = [s.strip() for s in getenv("LITESTAR_RELOAD_INCLUDES", "").split(",") if s] or None
+        reload_excludes = [s.strip() for s in getenv("LITESTAR_RELOAD_INCLUDES", "").split(",") if s] or None
```

Comment:
is this supposed to be `LITESTAR_RELOAD_INCLUDES` for exclude?

Reply:
Thanks for the review, it just stayed the same when copying, I'll change it now!)

And about the tests, I'll think about it

Reply:
You would also have to add these in https://github.com/litestar-org/litestar/blob/ca3659a3db5d9197b3341a38f54b7e0420f07424/docs/usage/cli.rst?plain=1#L127-L130 such that https://docs.litestar.dev/latest/usage/cli.html#cli-run-options shows the new env vars you have added. I am assuming others are ok with these names.

Reply:
> You would also have to add these in
> 
> https://github.com/litestar-org/litestar/blob/ca3659a3db5d9197b3341a38f54b7e0420f07424/docs/usage/cli.rst?plain=1#L127-L130
> 
> such that https://docs.litestar.dev/latest/usage/cli.html#cli-run-options shows the new env vars you have added. I am assuming others are ok with these names.

I have updated the documentation, can you check and give advice if there is anything else you need to add besides tests? I will add tests soon!)